soc/arm/microchip_mec/mec1501/power.c

```diff
@@ -50,7 +73,7 @@ static void z_power_soc_deep_sleep(void)
 	soc_deep_sleep_enable();
 
 	soc_deep_sleep_wait_clk_idle();
-	soc_deep_sleep_non_wake_en();
+	/* soc_deep_sleep_non_wake_en(); */
```

Comment:
Would commenting out the `soc_deep_sleep_non_wake_*()` affect deep sleep? Might want to wrap these under `DEBUG_SLEEP_ENTRY`. If these affect actual deep sleep, might want to do a SoC Kconfig so this can be turned on if desired.

Reply:
Incorrectly added commits for debug